xdsl/dialects/stencil.py

```diff
@@ -1225,7 +1226,9 @@ def matches(attr: TensorType[Attribute], other: Attribute) -> bool:
     def verify(self, attr: Attribute, constraint_context: ConstraintContext) -> None:
         ctx_attr = constraint_context.get_variable(self.name)
         if ctx_attr is not None:
-            if isa(attr, TensorType[Attribute]) and TensorIgnoreSizeConstraint.matches(
+            if isa(
+                attr, TensorType[Attribute]
```

Comment:
```suggestion
                attr, TensorType
```
unrelated but may as well change while we're here?